install_files/ansible-base/roles/restore/files/restore.py

```diff
@@ -35,18 +36,29 @@ def verify_args():
 def main():
     verify_args()
 
+    # Remove the /var/lib/tor/services directories to purge values that may have been
+    # generated by running the ansible playbooks
+    for d in ['journalist', 'source']:
```

Comment:
@heartsucker @emkll: for posterity can you explain why this change was necessary? Does the stanza below not overwrite the `/var/lib/tor/services` config files on disk in the correct locations?

Reply:
In retrospect, it does not seem strictly necessary, unless I am missing something: the tar extraction should squash existing values on the server

Reply:
I'm pretty sure V3 onion services add additional files, and I can't remember how `tar -x` works and if those extra files would cause bad behavior in the future.

Reply:
Based on the commit history and comments, it's not clear why this stanza is needed... it seems like a reasonable approach here is to determine if it's necessary. If it is, let's document why (at least in the PR). If we actually don't need this stanza, then we can just remove it in the spirit of reducing LOC to maintain.